test-bdd/testutil/helpers.go

```diff
@@ -35,6 +35,7 @@ import (
 type TemplateArguments struct {
 	ClusterName        string
 	KeyPairName        string
+	ProfileName        string
```

Comment:
We can hardcode ProfileName in the template, is there significance in requiring to export the name as a parameter for the test? Test look good otherwise, we just need some basic assertion after create/delete that will tell us the reconcile happened as we expected, since there are no Kubernetes resources created as a result, it could be something like "And a fargate profile <name> is (found|not found) - this will also be easier by having a predictable hard-coded name for the profile.

Reply:
👍